Assets/_Scripts/UI-scripts/PlayerBuffView.cs

```diff
@@ -10,6 +10,7 @@ public class PlayerBuffView : MonoBehaviour {
 	// Use this for initialization
 	void Start () {
 		image = GetComponent<Image> ();
+		playerId = transform.parent.transform.parent.GetComponent<UserInputManager> ().playerNum;
```

Comment:
This can obtain from GameStatus by Player Instance.
Should be called getPlayerID or something.
You may refer to PlayerNameView